### PR TITLE
update index

### PR DESCRIPTION
**Pull request by mforets**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -93,14 +93,14 @@ <h2 class="page-header"><i class="fa fa-cubes"></i> Projects</h2>
      <ul>
      <li><a href="https://github.com/JuliaReach/LazySets.jl/">LazySets.jl</a> A Julia package for calculus with convex sets.</li>
      <li><a href="https://github.com/JuliaReach/Reachability.jl/">Reachability.jl</a> Reachability and Safety of Nondeterministic Dynamical Systems.</li>
-     <li><a href="https://github.com/JuliaReach/MathematicalSystems.jl/">MathematicalSystems.jl</a> Systems definitions in Julia.</li>
```
Comment (schillic) [inline index.html]: There is a sentence in the paragraph above about `MathematicalSystems` which should be (re)moved I think.

File: index.html [hunk 2]
```diff
@@ -93,14 +93,14 @@ <h2 class="page-header"><i class="fa fa-cubes"></i> Projects</h2>
...
      <br>
      </ul>
 
 <p align="justify">
 The following component packages are developed and maintained by the JuliaReach org. 
 </p>
      <ul>
+     <li><a href="https://github.com/JuliaReach/MathematicalSystems.jl/">MathematicalSystems.jl</a> Systems definitions in Julia.</li>
```
Comment (schillic) [inline index.html]: I wonder if we should remove the "in Julia" everywhere because it is quite obvious.

File: index.html (hunk 1 above)
Comment (mforets) [reply index.html]: True; it is more of a (julian) cultural thing to use "... in Julia" , ".. for Julia..", "A Julia ..." in the package's description, and i like it, but we are not bound to that convention.

Comment (mforets) [reply index.html]: (*) this is seen by exploring a bit https://pkg.julialang.org/docs/

**Pull request merged**